components/Cards/SchoolCard/SchoolCard.css

```diff
@@ -65,6 +65,80 @@
   left: 100%;
 }
 
+.vetTecRibbon {
```

Comment:
The only good critique I can really add is toward reducing the amount of CSS being written.

It seems like there exists enough here to create a `.ribbon` class and then you can the exclusive subset of rules like:

```css
.ribbon .vetTech {}
.ribbon .giBill {}
.ribbon .dual {}
```

Reply:
Thank you for this idea. It certainly simplified the ribbons specific classes and will make it easy to add others or change them in the future.